framework/sandstone.cpp

```diff
@@ -3386,21 +3386,25 @@ int main(int argc, char **argv)
             return EXIT_SUCCESS;
         case one_sec_option:
             test_list_randomize = true;
+            test_selection_strategy = Repeating;
             sApp->use_strict_runtime = true;
```

Comment:
I guess these aren't needed any more as they will be set in the if statement below.

Reply:
This may appear redundant but the clause below is a guard clause to prevent ordered and alphabetical test ordering when the user explicitly asked for random.  The lines above are to determine what kind of random strategy will be used for these specific modes.  it is possible that we may change these modes to be NonRepeat while desiring the guard clause below to still be Repeat.

Now that I think about it, the default for test-list-randomize should probably be NonRepeat while --Xmin should probably be repeat.  this is based on the idea that --test-list-randomize would imply running the test list from start to finish in a random order, while the others are more of a weighted run list with no real end.

Reply:
I was referring to sApp->use_strict_runtime = true;

Reply:
This is still needed, otherwise the test list will continue to run to completion before it stops.
Without this option, the main loop will not check the duration often enough to end the run in the appointed time.